src/Okta.Idx.Sdk/IdxConfigurationValidator.cs

```diff
@@ -44,6 +44,24 @@ public static void Validate(IdxConfiguration configuration)
                     nameof(configuration.RedirectUri),
                     "Your Okta Application redirect URI is missing.You can find it in the Okta Developer Console in the details for the Application you created.");
             }
+
+            if (!IsValidUri(configuration.RedirectUri))
+            {
+                throw new UriFormatException($"Your Okta Application redirect URI is not a valid URI: {configuration.RedirectUri}");
+            }
+        }
+
+        private static bool IsValidUri(string value)
+        {
+            try
+            {
+                Uri uri = new Uri(value); // this will throw a UriFormatException on invalid value.
+                return true;
+            }
+            catch (UriFormatException)
+            {
+                return false;
```

Comment:
Why don't we throw directly from here since we're rethrowing the same exception afterward? Exceptions are expensive and we should try to avoid catch and rethrow.